vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/registry/LComponentManagementRegistry.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © 2019 XDEV Software (https://xdev.software)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package software.xdev.vaadin.maps.leaflet.registry;
+
+import java.io.Serializable;
+
+import com.vaadin.flow.component.page.PendingJavaScriptResult;
+
+import software.xdev.vaadin.maps.leaflet.base.LComponent;
+import software.xdev.vaadin.maps.leaflet.base.LComponentOptions;
+
+
```

Comment:
There needs to be some JavaDoc explaining that thing.

Reply:
Since this is such a core principle now in the library i think we should add some explanation and reasoning in the README about this registry.

Reply:
> There needs to be some JavaDoc explaining that thing.

Done

> Since this is such a core principle now in the library i think we should add some explanation and reasoning in the README about this registry.

I don't think we should write this into the README as a README usually doesn't represented detailed documentation.
I'm currently fine with the entry in the changelog, the JavaDoc on the class and the comments in the demo.